README.markdown

```diff
@@ -187,7 +187,15 @@ let colour = "red"
 
 ## Code Organization
 
-Use extensions to organize your code into logical blocks of functionality. Each extension should be set off with a `// MARK: -` comment to keep things well-organized.
+Use extensions to organize your code into logical blocks of functionality. Each extension should be set off with a the following comment to keep things well-organized.
+
+```
+    //==========================================================================
+    // MARK: - Something
+    //==========================================================================
+```
```

Comment:
Are we still using this kind of Marks?

I am not a super fan of it (specially now that we get to see "normal" marks in xcode minimap.